playground/didkey/src/main.rs

```diff
@@ -0,0 +1,9 @@
+use did_key::*;
+
+fn main() {
+    let key = DIDKey::new(DIDKeyType::Ed25519);
+    println!("{}", key.fingerprint());
+
+    let did_doc = key.to_did_document(CONFIG_LD_PUBLIC);
```

Comment:
Is it this easy?
Where does the config LD public come from?

Reply:
**Is it this easy?**

The `did:key` spec says that generating the did-document is just a function with the public key, which makes the `did:key`-method very simple, but very inflexible..

>The creation of a DID Document is also performed by taking the public key value and expanding it into DID Document.

Ref: https://w3c-ccg.github.io/did-method-key/#create


**Where does the config LD public come from?**

`CONFIG_LD_PUBLIC` is a constant impored from `did-key::*`